components/integrations/bots/bots.jsx

```diff
@@ -0,0 +1,206 @@
+                        then(() => {
+                            this.setState({loading: false});
+                        });
+                }
+            }
+        );
+    }
+
+    DisabledSection(props) {
+        if (!props.hasDisabled) {
+            return null;
+        }
+        const botsToDisplay = React.Children.map(props.disabledBots, (child) => {
+            return React.cloneElement(child, {filter: props.filter});
+        });
+        return (
+            <React.Fragment>
+                <div className='bot-disabled'>
+                    <FormattedMessage
+                        id='bots.disabled'
+                        defaultMessage='Disabled'
+                    />
+                </div>
+                <div className='bot-list__disabled'>
+                    {botsToDisplay}
+                </div>
+            </React.Fragment>
+        );
+    }
+
+    EnabledSection(props) {
+        const botsToDisplay = React.Children.map(props.enabledBots, (child) => {
+            return React.cloneElement(child, {filter: props.filter});
+        });
+        return (
+            <div>
+                {botsToDisplay}
+            </div>
+        );
+    }
+
+    render() {
+        const bots = Object.values(this.props.bots).sort((a, b) => a.username.localeCompare(b.username));
+        const botToJSX = (bot) => {
+            return (
+                <Bot
+                    key={bot.user_id}
+                    bot={bot}
+                    owner={this.props.owners[bot.user_id]}
+                    accessTokens={this.props.accessTokens[bot.user_id] || {}}
+                    actions={this.props.actions}
+                    team={this.props.team}
+                />
+            );
+        };
+        const enabledBots = bots.filter((bot) => bot.delete_at === 0).map(botToJSX);
+        const disabledBots = bots.filter((bot) => bot.delete_at > 0).map(botToJSX);
+
+        return (
+            <BackstageList
+                header={
+                    <FormattedMessage
+                        id='bots.manage.header'
+                        defaultMessage='Bot Accounts'
+                    />
+                }
+                addText={
+                    <FormattedMessage
+                        id='bots.manage.add'
+                        defaultMessage='Add Bot Account'
+                    />
+                }
+                addLink={'/' + this.props.team.name + '/integrations/bots/add'}
+                emptyText={
+                    <FormattedMessage
+                        id='bots.manage.empty'
+                        defaultMessage='No bot accounts found'
+                    />
+                }
+                helpText={
+                    <FormattedMessage
+                        id='bots.manage.help'
+                        defaultMessage='Use {botAccounts} to integrate with Mattermost through plugins or the API. Bot accounts are available to everyone on your server.'
```

Comment:
0/5 on changing this now, but this could use a `FormattedMarkdownMessage` to declare the link inline in the translation string

Reply:
I don't think we should be abusing markdown. It's likely way slower than this straight substitution.

Reply:
I think this is the intended use of `FormattedMarkdownMessage` when it was added, but I do agree that we tend to abuse Markdown in cases like this. Since the substitution doesn't make this sentence unreadable, it's probably better this way in this case.